docker/Dockerfile.updater

```diff
@@ -1 +1 @@
-FROM ghcr.io/dependabot/dependabot-updater/dependabot-updater@sha256:45fba063271b238fbb85479538d9ab8b4e27317674b98dc3e9519036816de3ba
+FROM ghcr.io/dependabot/dependabot-updater/dependabot-updater@sha256:fc30c3c4598f92943c461d04973b3ae9df3c0a09d757c0c467e2448d87a63d6a
```

Comment:
My intention is that this image, which is currently the most recent image for the updater, will be automatically updated by Dependabot once I merge a change to the updater. I plan to do this as soon as this PR is merged.